client/app/bundles/course/container/Breadcrumbs/sliders.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  RefObject,
+  UIEventHandler,
+  useLayoutEffect,
+  useRef,
+  useState,
+} from 'react';
+import { ChevronLeft, ChevronRight } from '@mui/icons-material';
+import { IconButton, Slide } from '@mui/material';
+
+interface UseSlidersHook {
+  ref: RefObject<HTMLDivElement>;
+  showStart: boolean;
+  showEnd: boolean;
+  handleScroll: UIEventHandler<Element>;
+  onClickStart: () => void;
+  onClickEnd: () => void;
+}
+
+export const useSliders = (): UseSlidersHook => {
```

Comment:
Interesting component! UI looks smooth but one may not know the implementation behind it is not as simple!